README.md

```diff
@@ -47,4 +47,11 @@ Run the .bat files to config the enviorment
     windows-dependencies\setenv.bat
 
 Step5:
+Install vcpkg to build all the third-party dependecies: go to vcpkg-scripts, read the Readme.md file for more instructions open a bash shell for MS Windows and execute the script to install vcpkg.

```

Comment:
Folder name is vcpkg-script, either change the folder name or correct here.

Not sure what you mean by bash shell, is it the command line you get on windows or windows powershell or even WSL.